display_list/dl_op_records.h

```diff
@@ -58,43 +58,45 @@ struct DLOp {
   static constexpr uint32_t kDepthInc = 0;
   static constexpr uint32_t kRenderOpInc = 0;
 
-  DisplayListOpType type : 8;
-  uint32_t size : 24;
+  explicit DLOp(DisplayListOpType type) : type(type) {}
+
+  const DisplayListOpType type;
```

Comment:
[Note to reviewers]
The elimination of the size field was the only change really needed in this file, but for some reason deleting that field caused clang-tidy to complain about the type field being uninitialized on every object below. So, I needed to add a parent initializer to every constructor below.

In the end, this is probably a good thing as it allowed me to make the type field const representing its important static role in the identity of the records.